Cesium.CodeGen/Extensions/TypeDefinitionEx.cs

```diff
@@ -17,6 +17,8 @@ internal static class TypeDefinitionEx
             name,
             MethodAttributes.Public | MethodAttributes.Static,
             returnType);
+
+        var _ = method.Body.Instructions; // Initalize instructions with empty collection to avoid NRE
```

Comment:
Why do we need this discard local variable?

Please clarify what does it mean the comment `// Initalize instructions with empty collection to avoid NRE` ?

Again typo `Initalize` -> `Initialize`